src/Proto.Actor/LocalContext.cs

```diff
@@ -124,18 +134,26 @@ public PID SpawnNamed(Props props, string name)
 
         public void SetBehavior(Receive receive)
         {
-            _behavior.Clear();            
+            _behavior = null;

```

Comment:
I think this is good, if we can clear out objects that are no longer used, we should do so 👍